### PR TITLE
Changed the function called cover to mincover. Tightened the bound for the fragmenter by adding shrinktransfer. Added comment.

### PR DESCRIPTION
**Pull request by brrmorre**
**Related issue**: 

**Type of change**: other enhancement

**Impact**: API addition (no impact on existing code)

**Development Phase**: implementation

**Release Notes**

Changed the function called cover to mincover.
Tightened the bound for the fragmenter by adding shrinktransfer.
Added comment.
Changed setName to name in the v2 constructor.
This is part of adding emits parameters to RC.

### CODE REVIEW

File: src/main/scala/amba/axis/Parameters.scala [hunk 1]
```diff
@@ -61,7 +61,7 @@ class AXISSlavePortParameters private (
   beatBytes.foreach { b => require(isPow2(b)) }
 
   val endDestinationId = slaves.map(_.destinationId).max + 1
-  val supportsCover = TransferSizes.cover(slaves.map(_.supportsSizes))
+  val supportsCover = TransferSizes.mincover(slaves.map(_.supportsSizes))
```
Comment (hcook) [inline src/main/scala/amba/axis/Parameters.scala:64]: so should this be `supportsMinCover` now?

File: src/main/scala/amba/axis/Parameters.scala [hunk 2]
```diff
@@ -146,7 +146,7 @@ class AXISMasterPortParameters private (
   beatBytes.foreach { b => require(isPow2(b)) }
 
   val endSourceId = masters.map(_.sourceId.end).max
-  val emitsCover = TransferSizes.cover(masters.map(_.emitsSizes))
+  val emitsCover = TransferSizes.mincover(masters.map(_.emitsSizes))
```
Comment (hcook) [inline src/main/scala/amba/axis/Parameters.scala:149]: so should this be `emitsMinCover` now?

File: src/main/scala/tilelink/Parameters.scala [hunk 3]
```diff
@@ -420,7 +420,7 @@ object TLSlaveParameters {
     address:       Seq[AddressSet],
     nodePath:      Seq[BaseNode]                = Seq(),
     resources:     Seq[Resource]                = Seq(),
-    setName:       Option[String]               = None,
+    name:       Option[String]                  = None,
```
Comment (hcook) [inline src/main/scala/tilelink/Parameters.scala:423]: let's keep the whitespace aligned here

**Comment by jackkoenig** [comment]
> **Impact:** API addition (no impact on existing code)

For the record this an API modification. I doubt any users are calling it but we need to be intentional on these kinds of things.

**Pull request merged**